src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps/Stick/ZHighStick.td

```diff
@@ -80,4 +80,43 @@ def ReplaceONNXLeakyReluPattern: Pat<
    (SameLayout $X, $stickout)]
 >;
 
+// Since zDNN does not support Sqrt(X), we calculate it by using zDNN-supported
+// operations, i.e. Exp and Log.
+// Formulas: `sqrt(X) = exp(log(x)/2)`
+def ReplaceONNXSqrtPattern: Pat<
+  (ZHighStickOp:$stick (ONNXSqrtOp (ZHighUnstickOp $X)), $layout),
+  (ZHighExpOp (ZHighDivOp (ZHighLogOp $X, (returnType $X)),
+                          (ZHighStickOp (GetConstantOfType<2> $X), $layout),
+                          (returnType $X))),
+  [(IsStaticShapeTensor $X), (SameLayout $X, $stick)]
+>;
+
+// Calulation of `1/sqrt(X)` or reciprocal square root is often found in
+// deep learning models, but zDNN does not support it. Thus, we rewrite it into
+// zDNN-supported operations.
+//
+// In this rewriting pattern we use:
+//   - `exp(log(x)/2)` for `sqrt(x)`, and 
+//   - `div(1,x) for `reciprocal`
+// So, `1/sqrt(X) = div(1, exp(log(x)/2))`
+//
+// The pattern
+//   zhigh.Stick (onnx.Reciprocal (onnx.Sqrt (zhigh.Unstick (%X))))
+// can be replaced by
+//   zhigh.Div (1, zhigh.Exp (zhigh.Div (zhigh.Log(%X), 2))
+//
```

Comment:
Can we use this equality?

1/(x^n) == x^(-n)

and thus 1/sqrt(n) = 1 / x^(1/2) = x ^ (-1/2).

using this with the log, we would get
1/sqrt(n) = exp( (-1/2) * log(x))

Also, is the `x / 2` as fast as the `0.5 * x` on the zAIU?

Reply:
Good suggestion! I will do some experiments to see which one is the best on the zAIU. Thanks!

Reply:
Did it worked?

Reply:
I am sorry that I haven't had chance to do experiments because the accelerator machine is having an issue creating a container. 

So I changed to use `1/sqrt(x)= exp (-0.5 * log(x))` in the code. In general, using `*` would be better than `/`. I think  the patch is ready.